From a4b33f4af3d7e99b15eae16ba972739dcf878349 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 27 Jan 2015 10:27:36 +0800
Subject: [PATCH] MLK-10244-10 ARM: dts: imx7d: add snvs-power key support

Enable rtc-snvs and snvs-power key, now we can use POWER key to wakeup
system.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit f292db590e0b01cf9580109c35d2a80a4b936c17)
---
 arch/arm/boot/dts/imx7d.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index a5de876..23d89bc 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -354,6 +354,14 @@
 				};
 			};
 
+			snvs-pwrkey@0x30370000 {
+				compatible = "fsl, imx7d-snvs-pwrkey", "fsl,imx6sx-snvs-pwrkey";
+				reg = <0x30370000 0x10000>;
+				interrupts = <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>;
+				fsl,keycode = <116>; /* KEY_POWER */
+				fsl,wakeup;
+			};
+
 			clks: ccm@30380000 {
 				compatible = "fsl,imx7d-ccm";
 				reg = <0x30380000 0x10000>;
-- 
1.8.0

